randomnumber/randomnumber/models.py

```diff
@@ -6,7 +6,7 @@
 
 class RandomNumber(models.Model):
     def get_random_number() -> int:
-        return random.getrandbits(64)
+        return random.getrandbits(random.randint(60, 64))
```

Comment:
This is to make overflows happen more often, right?

Reply:
Less often